kin-core/src/main/java/kin/core/KinClient.java

```diff
@@ -17,13 +16,12 @@
     private final List<KinAccountImpl> kinAccounts = new ArrayList<>(1);
 
     /**
-     * KinClient is an account manager for a single {@link KinAccount} on
-     * ethereum network.
+     * KinClient is an account manager for a {@link KinAccount}.
      *
      * @param context the android application context
-     * @param provider the service provider to use to connect to an ethereum node
+     * @param provider the service provider provides blockchain network parameters
```

Comment:
it not only provides the network parameters, right? Probably also handles all blockchain-related network stuff?